swagger.yml

```diff
@@ -1408,6 +1461,26 @@ definitions:
     required:
       - typ
 
+  angeforderteA2aKonten:
+    type: object
+    properties:
+      konten:
+        type: array
+        items:
+          $ref: '#/definitions/angefordertesA2aKonto'
+    required:
+      - konten
+      - format
```

Comment:
Das Format wollten wir doch jetzt nicht mehr in der API angeben oder? Scheint hier auch vergessen zu sein, denn oben in den Properties ist es schon weg.